### ISSUE TITLE
Add more help text for Cancel button on Confirm Save Mappings modal

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#682]
I think we need to remove the Cancel button on the Confirm Save Mappings dialog box. It puts the data into a weird state such that the records aren't matched, and can't be made to be matched automatically. So the user could potentially get their data into a state that they wouldn't be able to recover from very easily. 

![image](https://cloud.githubusercontent.com/assets/6314950/12570646/61a8cb82-c38e-11e5-88c3-1867a357a9eb.png)

@danielmcquillen and @StephenCzarnecki -- could you confirm (just with a comment) that you concur with this assessment, and then I will assign it to someone. 

thx

**Comment by RDmitchell** [comment]
Clicking the Cancel button does put you back into the Mapping screen, so maybe this is not such a problem?

**Comment by StephenCzarnecki** [comment]
The issue is that when this screen appears
![image](https://cloud.githubusercontent.com/assets/7605407/12599636/849e63ee-c448-11e5-82c8-d61f7b90703e.png)

The buildings have been added to the BuildingSnapshot table but no canonical building has been assigned and no merging has been done.  For example, if I use a small input file that only has one record in it by the time I get to that screen the BuildingSnapshot table looks like this in the database:

![image](https://cloud.githubusercontent.com/assets/7605407/12599765/6be1aaf4-c449-11e5-9d79-c4cd2c787f24.png)


Note how there are two records (67789 is the raw record from the file and 67790 is after the mapping has been done).  However canonical_building_id is blank and the canonicalbuilding table is empty:
![image](https://cloud.githubusercontent.com/assets/7605407/12599804/b0c76de8-c449-11e5-8464-f24d20f418bb.png)

It is only after hitting "Confirm mappings" that a record gets added to the canonicalbuilding table and the canonical_building_id is set.

So if the user hits cancel then the files they just added will only be visible on dialogs that deal with all building snapshots, not just the canonical ones.  One such example is the review mappings screen.  That shows non-canonical building snapshot records so these buildings will be visible then.

If the user hits cancel it is not unrecoverable.  They should be able to go into the review matches screen and match there at a later time.

I guess the reason for having the cancel button is written in the dialog.  Since the users cannot (currently) change things after matching it does seem fare to give them a warning and a way to stop the process if they think there might be an error.  So maybe some additional text in the dialog box explaining that if they cancel and navigate away from the screen in order to resume the current work they need to go back to the project and review mapping or matching for the file in question?

**Comment by RDmitchell** [comment]
This is no longer relevant. There is now a Dismiss button that puts you back in the Matching Review screen.
